parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -383,7 +383,7 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     {{else}}
     PRIVATE_IP=$(hostname -i | cut -d" " -f1)
     {{end}}
-    ETCD_CLIENT_PORT={{WrapAsVariable "masterEtcdClientPort"}}
```

Comment:
something funky in the state of Denmark, as this `ETCD_CLIENT_PORT` variable assignment shouldn't have been here. Probably legacy copy pasta.